x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -401,13 +400,26 @@ const getSavedObjectTypes = (
       },
     },
   },
+  [MESSAGE_SIGNING_KEYS_SAVED_OBJECT_TYPE]: {
+    name: MESSAGE_SIGNING_KEYS_SAVED_OBJECT_TYPE,
+    hidden: true,
+    namespaceType: 'agnostic',
+    management: {
+      importableAndExportable: false,
+    },
+    mappings: {
+      dynamic: false,
+      properties: {
+        private_key: { type: 'binary' },
+        public_key: { type: 'binary' },
+        passphrase: { type: 'binary' },
+      },
```

Comment:
Sorry if I wasn't clear in previous review. If you don't search on these fields, you **don't need** to specify them in the mapping, as long as the parent property is `dynamic: false`.

What I was expecting here is

```ts
mappings: {
      dynamic: false,
      properties: {},
}
```

(Unless there's a specific thing with binary fields I'm not aware of?)